indico/htdocs/sass/modules/contributions/_editor.scss

```diff
@@ -172,16 +172,6 @@ $editor-preview-bg-color: $light-gray;
     content: "\e14c";
 }
 
-.wmd-help-button {
```

Comment:
legacy file, will eventually be fully removed, but it was causing a duplicate "help" button in the wmd button bar so we just removed it from there already